src/main/java/build/buildfarm/cas/CASFileCache.java

```diff
@@ -2729,6 +2734,7 @@ void commit() throws IOException {
 
   @VisibleForTesting
   public static class Entry {
+    public static AtomicLong entryCount = new AtomicLong(0);
```

Comment:
This shouldn't be state maintained statically in the Entry list - all CASFileCaches will share this statistic if so. It needs to be a (final) instance member of the CFC, and maintained as such.